co-simulation/bundle/src/assembly/resources/scenarios/Town04/sumo/Town04.rou.xml

```diff
@@ -1,11 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- generated on 2023-05-23 13:36:19 by Eclipse SUMO netedit Version 1.16.0
+<!-- generated on 2023-11-06 14:44:39 by Eclipse SUMO netedit Version 1.18.0
 -->
 
 <routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/routes_file.xsd">
     <!-- Routes -->
-    <route id="r_0" edges="-7 -8 -9 -10 -11 -12 -13 -14 -36 -37 -38 -39 33 47 44 39 38 37 36 35"/>
+    <route id="r_0" edges="11 10 17"/>
+    <route id="r_1" edges="12 11 10 17"/>
+    <route id="r_2" edges="13 12 11 10 17"/>
     <!-- Vehicles, persons and containers (sorted by depart) -->
-    <vehicle id="v_0" depart="999.00" route="r_0"/>
-</routes>
\ No newline at end of file
+    <vehicle id="v_0" type="vehicle.carlamotors.carlacola" depart="0.00" route="r_2"/>
+    <vehicle id="v_1" type="vehicle.carlamotors.carlacola" depart="3.00" route="r_1"/>
+    <vehicle id="v_2" type="vehicle.carlamotors.carlacola" depart="8.00" route="r_0">
+    	<stop busStop="bs_2" duration="5.00"/>
+    </vehicle>
+    <vehicle id="v_3" type="vehicle.carlamotors.carlacola" depart="999.00" route="r_0"/>
```

Comment:
what is this one? seems like there are 4 vehicles?

Reply:
This one is used for keeping the scenario alive, if we only have three cars, the simulation will stop after the last car finishes, might not give us time for other Carma stuff we want to do, this is a safety